visualization/include/visualization/element.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <vector>
+#include <utility>
+#include <string>
+
+namespace visualization::builder {
+
+    class Element {
```

Comment:
`Element` is a bit too broad for the context here. This class defines an xml node and should reflect that